README.md

```diff
@@ -1,13 +1,12 @@
 # Happy Thoughts
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project is about happy-thoughts.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+In this project I used HTML, CSS, JS, React, API, BEM and NPM. I used npm to instal some libraries that were needed for my project like moment.js and fontawesome. When I had problem I used google and stackoverflow. If I had more time I would done black level :)
```

Comment:
Maybe come back to this part later and add some more info about the project ? :) 
Since recruiters will look over our github when we are looking for new jobs i think it can be good if the Read me maybe contains a little bit more info, so that someone who is not on the bootcamp can relate a little bit more?

Just a thought :)